### PR TITLE
Charlesmchen/tweak cells

### PR DESCRIPTION
**Pull request by charlesmchen-signal**
PTAL @michaelkirk 

Multiple small tweaks.

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m [hunk 1]
```diff
@@ -250,9 +248,11 @@ - (void)updateDateHeader
             [self.dateHeaderView autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.gutterLeading],
             [self.dateHeaderView autoPinTrailingToSuperviewMarginWithInset:self.conversationStyle.gutterTrailing],
             [self.dateHeaderView autoPinEdgeToSuperviewEdge:ALEdgeTop],
-            [self.dateHeaderView autoSetDimension:ALDimensionHeight toSize:self.dateHeaderHeight],
 
-            [self.messageBubbleView autoPinEdge:ALEdgeTop toEdge:ALEdgeBottom ofView:self.dateHeaderView],
+            [self.messageBubbleView autoPinEdge:ALEdgeTop
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m]: It seems like there's some subtlety behind this change. This looks equivalent, but less to the point. Why this change?

File: Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m [hunk 2]
```diff
@@ -94,26 +97,14 @@ - (void)loadForDisplayWithTransaction:(YapDatabaseReadTransaction *)transaction
...
-
-        [self.subtitleLabel autoPinEdge:ALEdgeTop
-                                 toEdge:ALEdgeBottom
-                                 ofView:self.titleLabel
-                             withOffset:self.subtitleVSpacing],
-        [self.subtitleLabel autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterLeading],
-        [self.subtitleLabel autoPinTrailingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterTrailing],
+        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeTop],
+        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeBottom],
+        [self.stackView autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterLeading],
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m]: For full width stuff, don't we want to pin to the edge here? not the margin?

File: Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m [hunk 3]
```diff
@@ -135,14 +126,9 @@ - (NSString *)subtitleForInteraction:(TSUnreadIndicatorInteraction *)interaction
...
-- (CGFloat)strokeHeight
-{
-    return 1.f;
-}
-
-- (CGFloat)subtitleVSpacing
+- (CGFloat)strokeThickness
 {
-    return 3.f;
+    return 1.f / UIScreen.mainScreen.scale;
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m]: TIOLI: Should we extract a free function like `CGHairlineWidth()`?

Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m]: Nice, done.

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m (hunk 1 above)
Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m]: Added a comment:

```
            // DO NOT pin to the bottom of dateHeaderView.
            //
            // Being a UIStackView, it doesn't reflect the spacing below the date
            // header contents.  Instead pin using dateHeaderHeight which includes
            // the spacing.
```

File: Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m (hunk 2 above)
Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m]: I was counting on `autoPinLeadingToSuperviewMarginWithInset ` being equivalent `autoPinEdgeToSuperviewEdge:ALEdgeLeading withInset:` since the cell margins are empty.  I preferred the former because it's more concise, but I've switched for clarity.
